### PR TITLE
[iOS] Try add xcodebuild for ios build workflow

### PR DESCRIPTION
**Pull request by ry-itto**
This reverts commit 2c264a68ab252a91056736b6de1e019c77776e16.

## Issue
- close #ISSUE_NUMBER

- This PR might be working... https://github.com/DroidKaigi/conference-app-2023/pull/693

## Overview (Required)
- 

## Links
- 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by ry-itto** [comment]
@tomoya0x00 
This seems like working. Please check this? 🙏🏼

File: app-ios/Modules/Package.swift [hunk 1]
```diff
@@ -106,6 +106,7 @@ var package = Package(
             name: "Sponsor",
             dependencies: [
                 "Assets",
+                "Component",
```
Comment (tomoya0x00) [inline app-ios/Modules/Package.swift:109]: I’m curious why did the build sometimes succeed without this line? 🤔 

I wonder if the builds of each dependency were running in parallel and `Component` was sometimes built before `Sponsor` was built by accident.

**Review by tomoya0x00 — APPROVED** [review]
LGTM 👍

File: app-ios/Modules/Package.swift (hunk 1 above)
Comment (ry-itto) [reply app-ios/Modules/Package.swift:109]: > Component was sometimes built before Sponsor was built by accident.

I think it's possible.
In last year's example, it was successful to build each module before building the app.

**Pull request merged**